src/sidebar/helpers/test/view-filter-test.js

```diff
@@ -80,11 +80,17 @@ describe('sidebar/helpers/view-filter', () => {
 
   describe('"any" field', () => {
     it('finds matches in any field', () => {
+      const defaultFields = {
+        text: '',
+        tags: [],
+        uri: 'https://example.com',
+        target: [{}],
+      };
```

Comment:
This adds some mandatory annotation fields, which this test's fixtures were previously missing.